src/webvr-manager.js

```diff
@@ -42,6 +42,10 @@ var Wakelock = require('./wakelock.js');
 function WebVRManager(renderer, effect, params) {
   this.params = params || {};
 
+  // Give a unique to ID to each manager.
+  this.prefix = 'webvr';
```

Comment:
Yes, it is a bit funky. However, having an auto-id could prevent issues
when future boilerplate versions allowed multiple instances of a manager on
the same page (e.g. in a VR gallery). Otherwise, events from the emitter
are going to broadcast to all instances, which may not be the desired
behavior (and DOM events have a huge overhead relative to the Emitter, so
they aren't a good listener substitute).

Several managers will share one Util, so the developer would need to
remember to explicitly add a specific Id for each manager instance.
Certainly possible, but providing an auto-id seems reasonable as a
'utility' function.

On Tue, Dec 29, 2015 at 11:08 AM, Boris Smus <notifications@github.com>
wrote:

> In src/webvr-manager.js
> <https://github.com/borismus/webvr-boilerplate/pull/89#discussion_r48562195>
> :
>
> > @@ -42,6 +42,10 @@ var Wakelock = require('./wakelock.js');
> >  function WebVRManager(renderer, effect, params) {
> >    this.params = params || {};
> >
> > +  // Give a unique to ID to each manager.
> > +  this.prefix = 'webvr';
>
> But why have the 'webvr' prefix be a member variable of the WebVRManager?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/borismus/webvr-boilerplate/pull/89/files#r48562195>.
>



-- 
============================================================
=================
*Dr. Pete Markiewicz*
*Email:* pindiespace@gmail.com
*Sustainable virtual design blog:*
http://sustainablevirtualdesign.wordpress.com
*Sustainability template:* http://greenboilerplate.com
*Teaching site: *http://plyojump.com <http://www.plyojump.com/>
*Buy my book! *- Millennials and the Popular Culture
*Lifecourse:* http://www.lifecourse.com/store/catalog/lca/mpc.html
* AmazonAuthor Page:*
http://www.amazon.com/Pete-Markiewicz/e/B0069VZ7BG/ref=ntt_athr_dp_pel_pop_2
*On that page, see my DARPA Grand Challenge 2005 entry video, "Team
Robomonster":*
*Team Robomonster *(DARPA 2005 Grand Challenge):
http://robotsthatjump.wordpress.com/category/team-robomonster/
*Indiespace *(1st to sell downloaded music, March 1994):
http://www.indiespace.com. Also see http://archive.org under
"Kspace.com" 1996 for our original site.
============================================================
=================